### DISCUSSION TITLE
Table 26, Panel 1

### DISCUSSION THREAD

**Discussion by stefanhenneking** [cssi-pi-community/2022-meeting#27] [Panel 1 roundtable (Monday morning)]
Table 26, Panel 1 discussion

**Comment by swuiuc** [comment]
Recruiting might be a bigger challenge than retention to improve the diversity of the CI workforce.

**Comment by stefanhenneking** [comment]
How can the NSF contribute to _recruiting_ a more diverse CI workforce? 
Maybe, recruiting becomes easier if there are more diverse _role models_ working in CI.

**Comment by swuiuc** [comment]
We need much more creative and disruptive ideas and work represented in NSF proposals and projects.

**Comment by amilworks** [comment]
I second the need for maintenance funding. **Developing** and **Maintaining** software should be two separate teams, and we need funding for both.
